src/asynch/ws.rs

```diff
@@ -1,44 +1,90 @@
-#[cfg(any(feature = "edge-net", feature = "embedded-svc"))]
+#[cfg(any(
+    feature = "edge-net",
+    feature = "embedded-svc",
+    feature = "embedded-svc-prost"
+))]
 pub use error::*;
 
 #[cfg(feature = "edge-net")]
 pub use edge_net_impl::*;
 
-#[cfg(feature = "embedded-svc")]
+#[cfg(any(feature = "embedded-svc", feature = "embedded-svc-prost"))]
 pub use embedded_svc_impl::*;
 
-#[cfg(any(feature = "edge-net", feature = "embedded-svc"))]
+#[cfg(any(
+    feature = "edge-net",
+    feature = "embedded-svc",
+    feature = "embedded-svc-prost"
+))]
 mod error {
     use core::fmt::{self, Debug, Display};
 
+    #[cfg(feature = "embedded-svc-prost")]
```

Comment:
Now here and further down on all `ProstError` code blocks you should conditionalize by `#[cfg(feature = "prost")]` instead.